### ISSUE TITLE
Blocks asset pipeline, if there are no .svg's to compile

### ISSUE THREAD

**Issue by buschtoens** [minutebase/ember-inline-svg#1]
After installing via `npm install --save-dev ember-inline-svg`, `ember serve` doesn't work anymore.

`ember serve` yields this output:
```
version: 0.1.4
Could not find watchman, falling back to NodeWatcher for file system events
Livereload server on port 35729
Serving on http://0.0.0.0:4200/
```

The build doesn't happen or doesn't get finished. If it was finished, this would be displayed:

```
version: 0.1.4
Could not find watchman, falling back to NodeWatcher for file system events
Livereload server on port 35729
Serving on http://0.0.0.0:4200/

Build successful - 1209ms.

Slowest Trees                  | Total          
-------------------------------+----------------
SassCompiler                   | 282ms          
EsnextFilter                   | 238ms          
EsnextFilter                   | 127ms          
Concat                         | 79ms   
```

**Comment by buschtoens** [comment]
This is "solved" by #7.

`broccoli-spelunk` didn't pass on Promise rejections, thus stalling the build. With #7 applied the following error is displayed and the build process is aborted:
```
Path or pattern "**/*.svg" did not match any files
```

However, this might not be the desireable behaviour. I would expect the build pipeline to just move on and emit an empty svg.js like:

```js
export default {}
```

**Comment by buschtoens** [comment]
This behaviour / error is caused by `broccoli-static-compiler` and is marked as *by design*.

There's a plugin called `broccoli-funnel` which doesn't have this problem. I fixed it and appended it to #7.

**Comment by buschtoens** [comment]
Fixed by #7.
